stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1558,20 +1571,24 @@ public enum DeviceKind {
 }
 
 internal extension _ExecutionContext {
-  var currentDeviceKind: DeviceKind {
+  var currentDevice: (DeviceKind, UInt32) {
```

Comment:
Two suggestions:
1. It would be nicer to label these tuple elements so that you can refer to elements by name (not by index).
2. From this file, I don't see any backend need for this number to be `UInt32` instead of `UInt`. Can we use `UInt` instead? Swift conventions prefer unsized integer types unless they are absolutely necessary.

```suggestion
  var currentDevice: (kind: DeviceKind, index: UInt) {
```

Reply:
Agree using tuple elt index is not great. This has been eliminated in this PR. It seems adding tuple elt labeling is not needed for code readability for now.

Agree with `UInt`. That has been addressed in Parker's prior PR.